.github/workflows/node-zxc-compile-application-code.yaml

```diff
@@ -191,7 +191,7 @@ jobs:
 
       - name: Gradle Dependency Scopes Check
         if: ${{ inputs.enable-dependency-check && steps.gradle-build.conclusion == 'success' && !cancelled() }}
-        run: ${GRADLE_EXEC} checkAllModuleInfo validatePomFiles --continue
+        run: ${GRADLE_EXEC} checkAllModuleInfo validatePomFiles --continue --no-build-cache
```

Comment:
This is an issue in the Dependency Analysis Plugin and only concerns the analysis tasks (not the building of our production code). You can only trigger it when you change the group of your components and do no other change. I yet have to create a reproducer and report this. Such a change is a rare situation.

If we move forward with this, I'll remove this `--no-build-cache` change again in a follow up PR.